enterprise/frontend/src/embedding-sdk/README.md

```diff
@@ -24,25 +25,48 @@ Features currently supported:
 - ability for the user to modify existing questions
 - theming with CSS variables
 - plugins for custom actions, overriding dashboard card menu items
-- subscribing to events
-- editing dashboards - requires upgrade to metabase v50
-- creating dashboards
-
-Features not yet supported:
-
-- letting users create new questions from scratch
+- subscribing to user or system events
+- creating and editing dashboards
+- create new questions from scratch and modifying existing questions
+
+## Known limitations
+
+* The SDK is currently only compatible with Metabase v50
```

Comment:
nit: let's use `-` for bullets here as well for consistency

Reply:
OK